code/style.css

```diff
@@ -1,7 +1,487 @@
+  border: 1px solid rgb(255, 147, 6);
+  transition: all 1s ease-in;
+}
+
+input:focus::placeholder {
+  color: white;
+  transition: all 1s ease-in;
+}
+
+.form__content__layout__row__room-selector {
+  font-size: 15px;
+  padding: 5px 5px 5px;
+  outline: none;
+}
+
+.form__content__layout__row__date-picker {
+  font-family: "Roboto", sans-serif;
+  font-size: 16px;
+  outline: none;
+  padding: 5px 5px 5px;
+}
+
+.form__content__layout__row--radio-btns {
+  display: grid;
+  grid-template-columns: 1fr auto;
+}
+
+.form__content__layout__row__radio-btn-container {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-template-rows: 1fr 1fr 1fr;
+  column-gap: 10px;
+  row-gap: 4px;
+}
+
+input[type="radio"] {
+  -webkit-appearance: none;
+  display: block;
+  width: 1em;
+  height: 1em;
+  border-radius: 50%;
+  border: 0.1em solid black;
+  margin: auto;
+  box-shadow: 1px 1px black;
+}
+
+input[type="radio"]:checked {
+  background-color: black;
+  box-shadow: none;
+  transition: all 0.5s ease-in;
+}
+
+.form__content__layout__user-agreement {
+  margin: 20px 0;
+  text-align: center;
+}
+
+input[type="checkbox"] {
+  -webkit-appearance: none;
+  display: block;
+  width: 1em;
+  height: 1em;
+  border: 0.1em solid black;
+  margin: auto;
+  box-shadow: 1px 1px black;
+  margin-top: 10px;
+}
+
+input[type="checkbox"]:checked {
+  box-shadow: none;
+  background-color: black;
+  transition: all 0.5s ease-in;
+}
+
+.form__content__layout__btn {
+  background-color: rgb(0, 0, 0, 0);
+  border: rgb(0, 0, 0) solid 0.5px;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 20px 10% 20px 10%;
+  width: 200px;
+  border-radius: 40px;
+  font-weight: 500;
+  text-decoration: none;
+  box-shadow: 2px 2px black;
+}
+
+.form__content__layout__btn:hover {
+  cursor: grab;
+  transition: all 1s ease-in;
+  box-shadow: none;
+}
+
+.form__posters {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.form__posters img {
+  width: 80%;
+  margin-top: 40px;
+}
+
+/* ------------- Flex containers----------------  */
+
+.container__accordion-contact {
+  display: flex;
+}
+
+.content__accordion-contact {
+  display: flex;
+  flex-direction: column;
+}
+
+/* ------------- ACCORDION----------------  */
+
+.accordion {
+  background-color: rgb(211, 209, 209);
+  padding: 30px 10% 50px 10%;
+}
+
+.accordion__question {
+  display: grid;
+  background-color: rgb(156, 212, 214);
+  font-weight: 500;
+  padding: 10px;
+  border: 0.5px solid black;
+  grid-template-columns: 1fr auto;
+}
+
+.accordion__answer {
+  background-color: rgb(228, 228, 228);
+  max-height: 0px;
+  border-right: 0.5px solid black;
+  border-left: 0.5px solid black;
+  overflow: hidden;
+  padding: 0;
+}
+
+.active + .accordion__answer {
+  max-height: 100%;
+  padding: 10px;
+  transition: all 400ms ease-in-out;
+}
+
+.accordion__question__arrow {
+  color: black;
+}
+
+.accordion__answer--last {
+  border-bottom: 0.5px solid black;
+}
```

Comment:
The accordion looks amazing. Loved the different colours for question and answer and the effect when opening it.

Reply:
Thanks for all of your comments, well needed boost and stuff to think about. Been struggeling with the long names so I really understand what you mean there.